lib/safira/accounts/user.ex

```diff
@@ -33,7 +33,7 @@ defmodule Safira.Accounts.User do
     |> downcase_email
     |> validate_required([:email, :password, :password_confirmation])
     |> validate_length(:email, min: 5, max: 255)
-    |> validate_format(:email, ~r/\A[^@\s]+@[^@\s]+\z/)
+    |> validate_format(:email, ~r/\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/)
```

Comment:
I think we could get away with not validating email. That standard is 3 pages long, no way we can write a sane regex for that.